refinery/core/models.py

```diff
@@ -1845,6 +1845,13 @@ class ExtendedGroup(Group):
     can_edit = False
     manager_group_uuid = None
 
+    def clean(self):
+        # Don't allow only white spaces.
+        if len(self.name) == 0:
+            logger.error("Group name cannot be empty.")
+            raise ValidationError("Group name cannot be empty")
+        return self.name.strip()
```

Comment:
The clean() method should not return any values: https://docs.djangoproject.com/en/1.7/ref/models/instances/#django.db.models.Model.clean